tools/publisher/src/subcommand/fix_manifests.rs

```diff
@@ -29,7 +31,11 @@ pub async fn subcommand_fix_manifests(mode: Mode, location: &str) -> Result<()>
     let manifest_paths = discover_package_manifests(location.into()).await?;
     let mut manifests = read_manifests(Fs::Real, manifest_paths).await?;
     let versions = package_versions(&manifests)?;
+
+    validate::pre_validate_manifests(&versions)?;
```

Comment:
might name these something to clarify what they are pre and post. "pre_validate_manifests" sounds like you're doing something before validation, not that you're validating the state of manifests, or you could update to be something like:

```
validate_manifests(&versions, |&mut manifests| fix_manifests(...))
```

Reply:
Renamed the functions to clarify their intent.